### PR TITLE
[th2-2334] prepare test build

### PR DESCRIPTION
**Pull request by Nikita-Smirnov-Exactpro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -119,6 +119,12 @@ spec:
 
 ## Release notes
 
+### 3.9.0
+
++ Embedded sailfish service based on MINA decodes message as sender during send.
```
Comment (jean-michel-perli-sonsin) [inline README.md:124]:
```suggestion
+ Embedded sailfish service based on MINA decodes the message as sender during sending.
```

File: README.md [hunk 2]
```diff
@@ -119,6 +119,12 @@ spec:
...
+  This approach is important for protocols where pair of messages have the same protocol message type and different structures depending on direction.
```
Comment (jean-michel-perli-sonsin) [inline README.md:125]:
```suggestion
  This approach is important for protocols in which a pair of messages have the same protocol message type and different structures depending on the direction.
```

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]
Hello, please see if you agree with the changes proposed below:

README.md

Line 40 - You can add the ability to connect to a target system by implementing your own service **to** the Sailfish format and putting it **together with** its configuration to the correct places **into** the base image.

Line 42 - You need to **perform** the following steps:

Line 68 - Create **your** own image based on the current one and put all **the** files **in** the correct places in the base image:

Line 71 - If you use the _plugin_alias_ and _name_ different from _th2_service_ in the VERSION file correct the _th2_service_ folder name according to **the** value **that** you **are using**.

Line 73 - Artifact with the service(s) implementation(s) and all its dependencies should be put **into** the following directory

Line 74 - The configuration file created on the step 2 should be put **into** the following directory

Line 75 - The _VERSION_ file created on step 3 should be put **into** the following directory

Line 126/132 - Netty services do not copy metadata to the `IMessage` when sending one. **This problem was fixed and now they copy metadata**.

Line 149/155 - **An alert is sent** if **it gets an** ErrorMessage when sending raw message

Line 154/160 - resets embedded log4j configuration before configuring **it** from a file

In case of any question, please let me know.
Thanks!

**Review by OptimumOpium — APPROVED** [review]

**Review by jean-michel-perli-sonsin — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -65,14 +65,14 @@ This file must contain:
     version: 3.2.0.0
     core_version: 3.2.0
     ```
-4. Create you own image based on the current one and put all files to the correct places in the base image:
+4. Create you own image based on the current one and put all the files in the correct places in the base image:
     + Create the following directory - **${workspace}/plugins/th2_service**.
     _**${workspace}**_ - it is a folder from the "Connect" configuration.
-    If you use the _plugin_alias_ and _name_ different from _th2_service_ in the VERSION file correct the _th2_service_ folder name according to value you use.
+    If you use the _plugin_alias_ and _name_ different from _th2_service_ in the VERSION file correct the _th2_service_ folder name according to the value that you are useing.
```
Comment (jean-michel-perli-sonsin) [inline README.md:71]:
```suggestion
    If you use the _plugin_alias_ and _name_ different from _th2_service_ in the VERSION file correct the _th2_service_ folder name according to the value that you are using.
```

**Review by jean-michel-perli-sonsin — DISMISSED** [review]

**Review by jean-michel-perli-sonsin — APPROVED** [review]

**Review by OptimumOpium — APPROVED** [review]

**Comment by Nikita-Smirnov-Exactpro** [comment]
Thank you for your revive. It has been moved to https://github.com/th2-net/th2-conn/pull/64

**Pull request closed without merging**